### DISCUSSION TITLE
Private instance and privacy

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searxng#1493] [Q&A] [answered]
So I have just sétup my private instance with Docker, of course I trust the owner, me😁, but I am aware that search engines still see my IP and that SearxNG can be configured to use Tor. So my question is: what is, in terms of privacy only, the benefit of using SearxNG +Tor instead of simply, say, googling with Tor? Wouldn't Tor alone make tracking and profiling useless for identifying you?

**Comment by return42** [comment, marked as answer]
On public instances we recommend to register at [searx.space](https://searx.space/) to mix up your queries with others .. but you asking for private instances with tor:

> googling with Tor? 

Your WEB-Browser is not connected to e.g. google .. and SearXNG crawls several search engines in one (google, ddg, bing, ..).

**Reply by ghost** [reply]
Thanks, ho do I register for Searx.space?

**Reply by return42** [reply]
Read first section ["For SearXNG instance operators"](https://github.com/searxng/searx-instances/blob/master/README.md) in the README of the https://github.com/searxng/searx-instances project.

**Reply by ghost** [reply]
Thanks again, even if I only host my instance locally (connecting to localhost:8080) without Tor or VPNs my IP is exposed anyway, right?

**Reply by mrpaulblack** [reply]
IF you host locally without any proxy outgoing engines request are done with your public IP, so its exposed yes.
